src/main/java/com/github/manolo8/darkbot/core/objects/swf/SwfPtrCollection.java

```diff
@@ -27,16 +27,36 @@
      */
     public abstract long getPtr(int i);
 
-    public void forEachIndexed(Consumer<Long> consumer) {
+    /**
+     * Iterates over pointers in collection
+     * starting from <b>last iterated + 1</b>
+     * <p>
+     * For example, if collection wasn't changed,
+     * order and pointers are the same,
+     * consumer wont be executed even once.
+     *
+     * @param consumer to execute
+     */
+    public void forEachMemorized(Consumer<Long> consumer) {
```

Comment:
maybe `forEachNewValue` or something like that? i don't like using the word `new` cause it's a java keyword with a different meaning, but `forEachMemorized` isn't very explicit either

Reply:
`forEachIncremental` may be a good name for it